### PR TITLE
fix remote grid filter

### PR DESCRIPTION
**Pull request by xalopp**
This fixes the remote filtering of the store and adds remote filtering to the example application.

@mmoll @andythedandy @elger @danielhallmann @periklis  could you please review this commit.

### CODE REVIEW

File: packages/mf-grid-filter-form/examples/Readme.md [hunk 1]
```diff
@@ -4,6 +4,17 @@ Following examples are available:
...
 - [Simple Grid](simple-grid/index.html) is a simple application, that uses the plugin feature.
+  You can also have a look at this example by using the PHP built-in web server:
+  
+In your project root directory (ExtJsPackages)
+
+```sh
+    $ php -S localhost:65432
+```
+
+An open the [Simple Grid Example](http://localhost:65432/packages/mf-grid-filter-form/examples/simple-grid/index.html).
```
Comment (elger) [inline packages/mf-grid-filter-form/examples/Readme.md]: Think this should be "And open ..." instead of "An open ..."

**Comment by elger** [comment]
:+1:

File: packages/mf-grid-filter-form/examples/Readme.md (hunk 1 above)
Comment (xalopp) [inline packages/mf-grid-filter-form/examples/Readme.md]: 100% true

File: packages/mf-grid-filter-form/examples/simple-grid/remotenumber.php [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+$dataFields = ['id', 'name', 'description'];
+
+$totalCount = count($data);
+
+$start = array_key_exists('start', $_GET) ? $_GET['start'] : 0;
+$limit = array_key_exists('limit', $_GET) ? $_GET['limit'] : 5;
+if (array_key_exists('filter', $_GET)) {
+    // [{"operator":"like","value":"Value","property":"name"}]
+    // [{"operator":"like","property":"id","value":"1"},{"operator":"like","property":"name","value":"F"},{"operator":"like","property":"description","value":"item"}]
```
Comment (mmoll) [inline packages/mf-grid-filter-form/examples/simple-grid/remotenumber.php]: comment? :speech_balloon:

File: packages/mf-grid-filter-form/examples/simple-grid/remotenumber.php [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            }
+
+            return $isMatch;
+        }
+    );
+}
+
+$numbers = $data;
+
+// sort:[{"property":"description","direction":"DESC"}]
```
Comment (mmoll) [inline packages/mf-grid-filter-form/examples/simple-grid/remotenumber.php]: comment? :speech_balloon:

**Comment by mmoll** [comment]
tested, works for me, :+1:
minor nitpick about leftover comments.

File: packages/mf-grid-filter-form/examples/simple-grid/remotenumber.php (hunk 2 above)
Comment (andythedandy) [inline packages/mf-grid-filter-form/examples/simple-grid/remotenumber.php]: Seal of approvement

**Pull request merged**